packages/admin/cms-admin/src/common/MasterMenu.tsx

```diff
@@ -80,9 +88,10 @@ export interface MasterMenuProps {
 
 export function useMenuFromMasterMenuData(items: MasterMenuData): MenuItem[] {
     const isAllowed = useUserPermissionCheck();
-    const checkPermission = (item: MasterMenuItem): boolean => !item.requiredPermission || isAllowed(item.requiredPermission);
+    const checkPermission = (item: MasterMenuItemRoute | MasterMenuItemAnchor | MasterMenuItemCollapsible | MasterMenuItemGroup): boolean =>
+        !item.requiredPermission || isAllowed(item.requiredPermission);
 
-    const mapFn = (item: MasterMenuItem): MenuItem => {
+    const mapFn = (item: MasterMenuItemRoute | MasterMenuItemAnchor | MasterMenuItemCollapsible | MasterMenuItemGroup): MenuItem => {
```

Comment:
Why do you need to change the types here? 🤔

Reply:
I need to for TypeScript because I added the icon property to the `MasterMenuItem` type for all except `MasterMenuItemGroup`